src/webots/nodes/WbGps.cpp

```diff
@@ -307,8 +307,9 @@ bool WbGps::refreshSensorIfNeeded() {
     mMeasuredPosition[2] = altitude;
   }
 
+  mSpeedVector = (t - mPreviousPosition);
```

Comment:
`mSpeedVector` should be computed in meters per second.

Reply:
Doh indeed.